src/Reflection/ReflectionType.php

```diff
@@ -86,7 +86,7 @@ public function __toString()
                 return 'bool';
             case $this->type instanceof Types\Float_:
                 return 'float';
-            case $this->type instanceof Types\Void:
+            case $this->type instanceof Types\Void_:
```

Comment:
This kind of change probably needs a test. When would this crash, and does the coverage reach it at the moment?